### PR TITLE
K8s docs etcd updates

### PR DESCRIPTION
**Pull request by evilnick**
## Done

- Update backup page to highlight v2 and v3 etcd snapshots
- Redirect all upgrading pages explicitly to the backups page
- Remove some outdated charm pages (more will follow once they are delinked - these charm docs are now in charmhub
- Update link for MAAS docs

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

**Comment by petesfrench** [comment]
Code looks good, but not sure how I am supposed to QA this?

**Comment by carkod** [comment]
> Code looks good, but not sure how I am supposed to QA this?

It's the same as discourse documentation, if you want to review `templates/kubernetes/docs/1.23/upgrading.md`

Visit https://ubuntu-com-12926.demos.haus/kubernetes/docs/1.23/upgrading

File: templates/kubernetes/docs/backups.md [hunk 1]
```diff
@@ -17,19 +17,21 @@ The state of your **Kubernetes** cluster is kept in the **etcd** datastore.
...
+by native Kubernetes. Various third party solutions are available - please refer to their own
+documentation for details.
 
 ## Creating an **etcd** snapshot
 
 <div class="p-notification--caution is-inline">
   <div markdown="1" class="p-notification__content">
     <span class="p-notification__title">Warning:</span>
-    <p class="p-notification__message">Snapshots can only be restored on the **same version of etcd**.</p>
+    <p class="p-notification__message">Snapshots can only be restored on the **same major version of etcd**.</p>
```
Comment (petesfrench) [inline templates/kubernetes/docs/backups.md:29]:
```suggestion
    <p class="p-notification__message">Snapshots can only be restored on the <em>same major version of etc</em>.</p>
```
This is not rendered as markdown so needs to use html tags

**Comment by petesfrench** [comment]
For the pages that are being deleted, is that information be displayed somewhere else? It might be worth setting up some redirects for it

File: templates/kubernetes/docs/1.23/upgrading.md [hunk 2]
```diff
@@ -410,7 +381,7 @@ juju run-action kubernetes-worker/1 upgrade
 
 ## Upgrading the Machine's Series
 
-All of the charms support [upgrading the machine's series via Juju](https://juju.is/docs/upgrading-series).
+All of the charms support [upgrading the machine's series via Juju](https://juju.is/docs/olm/manage-machines#heading--upgrade-the-ubuntu-series-of-a-machine).
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.23/upgrading.md]: Yes, i see the heading has been changed again. Updated in various places now

File: templates/kubernetes/docs/1.23/upgrading.md [hunk 3]
```diff
@@ -490,6 +461,8 @@ kube-system                       monitoring-influxdb-grafana-v4-65cc9bb8c8-mwvc
 [blue-green]: https://martinfowler.com/bliki/BlueGreenDeployment.html
 [validation]: /kubernetes/docs/validation
 [supported-versions]: /kubernetes/docs/supported-versions
+[juju-controller-upgrade]: https://juju.is/docs/olm/upgrade-models
+[bundle-repo]: https://github.com/charmed-kubernetes/bundle/tree/main/releases
```
[the remark this answers is not in the document]
Comment (evilnick) [reply templates/kubernetes/docs/1.23/upgrading.md:465]: It's autogenerated from a template so sometimes we end up with links defined that aren't used

**Comment by evilnick** [comment]
I think that's all the things now

RE deleted pages - they are no longer linked from anything - they were only ever linked in our docs from other pages in our docs so it shouldn't be an issue but I will submit some redirects when I kill the rest of them. The content is all in Charmhub now

**Pull request merged**